mlir/include/mlir/Conversion/GENToSPIRV/GENToSPIRV.h

```diff
@@ -23,8 +23,6 @@ class RewritePatternSet;
 namespace GEN {
 void populateGENToSPIRVPatterns(SPIRVTypeConverter &typeConverter,
                                 RewritePatternSet &patterns);
-
-std::unique_ptr<OperationPass<>> createConvertGENToSPIRVPass();
```

Comment:
Do we still get an auto-generated `createConvertGENToSPIRVPass` function or should we manually add one that takes the options as an argument?

Reply:
Auto generated :+1: